site/components/pages/Resources/Community/index.tsx

```diff
@@ -182,34 +182,34 @@ export const Community: NextPage<Props> = ({}) => {
               <Trans>Join Our Discord</Trans>
             </Text>
           </div>
-        </div>
-        <div className={styles.joinDiscord}>
-          <div className="joinDiscord_row1">
-            <Text t="h2" as="h2" align="center">
-              <Trans>Discord</Trans>
-            </Text>
-            <Text
-              t="body3"
-              color="lighter"
-              align="center"
-              className="padding20"
-            >
-              <Trans>
-                KlimaDAO harnesses the power of cryptocurrency, blockchain and
-                smart contracts to create
-              </Trans>
-            </Text>
-            <DiscordButton />
-          </div>
-          <div className="joinDiscord_row2">
-            <div className="joinDiscord_dummy">
-              <Image
-                src={screenShotDesktop}
-                width={764}
-                height={600}
-                alt="Screen shot of Discord"
-                layout="responsive"
-              />
+          <div className={styles.joinDiscord}>
+            <div className="joinDiscord_row1">
+              <Text t="h2" as="h2" align="center">
+                <Trans>Discord</Trans>
+              </Text>
+              <Text
+                t="body3"
+                color="lighter"
+                align="center"
+                className="padding20"
+              >
+                <Trans>
+                  KlimaDAO harnesses the power of cryptocurrency, blockchain and
+                  smart contracts to create
+                </Trans>
+              </Text>
+              <DiscordButton />
+            </div>
+            <div className="joinDiscord_row2">
+              <div className="joinDiscord_dummy">
+                <Image
+                  src={screenShotDesktop}
+                  width={804}
```

Comment:
hmm on my viewport it looks like the image is never larger than 560 px wide. We can save some bandwidth $ by reducing this

Reply:
I reduces the image now https://github.com/KlimaDAO/klimadao/pull/151
The image is now 64KB which is fine IMO
Changing the width and height to 560 x 415.82 made it look even blurrier again in my opinion. 😭  Why NextJS?